### PR TITLE
Admin pages

### PR DESCRIPTION
**Pull request by giuliamml**
WIP for issues:

- #47 - about page on `Navbar`
- #30 - list of all chains with info (page will be visible only to Admin when security will be implemented. 
   To view: http://localhost:3000/chains  and  http://localhost:3000/chains/:chainId/information)

### CODE REVIEW

File: frontend/src/pages/ChainsList.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  }, []);
+
+  const sortArray = () => {
+    if (chains == null) {
+      return chains;
+    } else {
+      let sortedArray = chains.sort((a, b) => a.name.localeCompare(b.name));
+      setSortedChains(sortedArray);
+      return sortedArray;
+    }
```
Comment (timstokman) [inline frontend/src/pages/ChainsList.js:35]: Why does this return anything? I wouldn't even make this a function to be honest.

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (timstokman) [inline yarn.lock:4]: I don't think this should've been checked in.

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules
+node_modules
```
Comment (timstokman) [inline .gitignore:2]: Why this double? Also, if you've had to delete the .env file, add it in .gitignore.

File: frontend/src/pages/ChainsList.js [hunk 4]
```diff
@@ -17,56 +17,45 @@ import { Typography } from "@material-ui/core";
...
   const [chains, setChains] = useState();
-  const [sortedChains, setSortedChains] = useState();
 
   //TODO get host details and render infos
 
   useEffect(() => {
-    getChains().then((response) => setChains(response));
+    getChains().then((response) => {
+      let sortedChains = response.sort((a, b) => a.name.localeCompare(b.name));
+      return setChains(sortedChains);
```
Comment (timstokman) [inline frontend/src/pages/ChainsList.js:26]:
```
let chains = await getChains();
let sortedChains = chains.sort((a, b) => a.name.localeCompare(b.name));
setChains(sortedChains);
```

File: .vscode/settings.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
+}
```
Comment (timstokman) [inline .vscode/settings.json:3]: Don't check in vscode settings

Comment (giuliamml) [reply .vscode/settings.json]: Removed vscode settings

**Review by timstokman — APPROVED** [review]

**Pull request merged**